src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -147,6 +147,26 @@ public override void VisitPointerType(IPointerTypeSymbol symbol)
             AddPunctuation(SyntaxKind.AsteriskToken);
         }
 
+        public override void VisitFunctionPointerType(IFunctionPointerTypeSymbol symbol)
```

Comment:
>VisitFunctionPointerType [](start = 29, length = 24)

Are we testing this?

Reply:
No, I just added it for ease of debugging (GetDebuggerDisplay uses it). I'll add a prototype, as the formatting is kinda wonky and I'm not planning on addressing this until I address the public API in general.

---
In reply to: [374345748](https://github.com/dotnet/roslyn/pull/41340#discussion_r374345748) [](ancestors = 374345748)